docs/source/rst_snippets/scancode-reindex-licenses.rst

```diff
@@ -1,23 +1,23 @@
 ``scancode-reindex-licenses`` Usage
 -----------------------------------
 
-Usage: ``scancode-reindex-licenses [OPTIONS]``
```

Comment:
Isn't CLI scripts/othe CLI commands best be wrapped by `` so they are highlighted differently?

Reply:
This is not addressed yet, can you revert the changes?